src/AlcoholTool.cpp

```diff
@@ -0,0 +1,123 @@
+/*
+ * AlcoholTool.cpp is part of Brewtarget and was written by Ryan Hoobler
```

Comment:
The copyright text is a bit funky versus what we normally use. How would that be updated when someone else modifies it? I'm not the one who wrote the original statement @rocketman768